pollux/lib/anoncreds/src/main/scala/io/iohk/atala/pollux/anoncreds/AnoncredLib.scala

```diff
@@ -83,16 +83,16 @@ object AnoncredLib {
           credentialOffer, // CredentialOffer credential_offer
         )
 
-    CreateCrendentialRequest(credentialRequest.getRequest(), credentialRequest.getMetadata())
+    AnoncredCreateCrendentialRequest(credentialRequest.getRequest(), credentialRequest.getMetadata())
```

Comment:
I see there are a lot of classes that start with the prefix `Anoncred`
I would propose to rename it (not now, but in some next PR with refactoring) to `Anoncreds` according to the spec name https://hyperledger.github.io/anoncreds-spec/

...and, probably, move all these classes to the package (package object) Anoncreds. From my point of view, it will make the name a little bit shorter :)

Reply:
Fixed here https://input-output.atlassian.net/browse/ATL-6566